src/components/action/action.tsx

```diff
@@ -295,7 +278,6 @@ export class Action
       buttonId,
       messages
     } = this;
-
     const ariaLabel = `${label || text}${indicator ? ` (${messages.indicator})` : ""}`;
```

Comment:
are parantheses in `(${messages.indicator})` intentional ?

Reply:
@driskull

Reply:
Yes, they are intentional.